odl/tomo/geometry/conebeam.py

```diff
@@ -184,22 +234,9 @@ def rotation_matrix(self, angle):
 
         return cos_ang * id_mat + (1. - cos_ang) * dy_mat + sin_ang * cross_mat
 
-    def __repr__(self):
-        """Return ``repr(self)``"""
-        inner_fstr = '{!r}, {!r}, src_radius={}, det_radius={}'
-        if self.has_motion_sampling:
-            inner_fstr += ',\n agrid={agrid!r}'
-        if self.has_det_sampling:
-            inner_fstr += ',\n dgrid={dgrid!r}'
-        inner_str = inner_fstr.format(self.motion_params, self.det_params,
-                                      self.src_radius, self.det_radius,
-                                      agrid=self.motion_grid,
-                                      dgrid=self.det_grid)
-        return '{}({})'.format(self.__class__.__name__, inner_str)
 
-
-class ConeFlatGeometry(ConeBeamGeometry):
-    """Cone beam geometry in 3d with flat detector.
+class CircularConeFlatGeometry(ConeBeamGeometry, AxisOrientedGeometry):
```

Comment:
This class is pretty much the same as the helical class except with `pitch=0`. Why not making this a subclass of the helical class?